packages/moda-sdk/src/profile/createProfile.test.ts

```diff
@@ -0,0 +1,51 @@
+import { createProfile } from './createProfile';
+import { metadataUri, profileAddress } from './testing/constants.ts';
+import { mockConfig } from './testing/mockConfig.ts';
+
+describe('createProfile', () => {
+  it('Returns a txHash', async () => {
+    const result = await createProfile({
+      config: mockConfig,
+      profileAddress,
+      metadataUri,
+    });
+
+    expect(result.ok).toBe(true);
+    if (result.ok) {
+      expect(result.value).toBe('txHash');
+    } else {
+      fail('Expected result to be successful');
+    }
+  });
+
+  it('Should fail if config is not verifiable', async () => {
+    const result = await createProfile({
+      config: { publicClient: undefined, walletClient: undefined },
+      profileAddress,
+      metadataUri,
+    });
+
+    expect(result.ok).toBe(false);
+    if (!result.ok) {
```

Comment:
This could be re-written as:

```ts
if (result.ok) {
  fail('Expected error result');
  return;
}

expect(result.error).toBe(ConfigError.PublicClientUndefined);
```

Reply:
(same as for the other tests. Line 28 is redundant

Reply:
Let's also do a test for when Public Client is valid and WalletClient is not